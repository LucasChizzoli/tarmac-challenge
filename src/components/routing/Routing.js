import React from 'react';
import { Switch, Route } from 'react-router-dom';
import Grid from '../grid/Grid';
import Detail from '../detail/Detail';

const Routing = () => {
	return (
		<Switch>
			<Route exact path="/" component={Grid} />
			<Route path="/member/:id" component={Detail} />
		</Switch>
	)
};

export default Routing;