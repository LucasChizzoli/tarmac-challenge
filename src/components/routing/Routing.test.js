import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Routing from './Routing';
import '../../EnzymeSetup.js';

describe('<Routing />', () => {
  describe('render()', () => {
    it('renders the component', () => {
      const wrapper = shallow(<Routing />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});