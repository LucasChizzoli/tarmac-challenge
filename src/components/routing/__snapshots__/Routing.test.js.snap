// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routing /> render() renders the component 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/member/:id"
  />
</Switch>
`;
